wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

```diff
@@ -0,0 +1,56 @@
+
+The _beatmap veto_ allows a [Beatmap Nominator](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators) to withhold a beatmap from Qualification if they feel there are significant issues regarding beatmap quality which make it unfit for the Ranked section.
+
+Seeking further discussion or clarification over any kind of quality issues which you feel need to be addressed before the map can proceed with Qualification is one of your main responsibilities. 
+
+Any Beatmap Nominator may place a beatmap veto, provided they are formally qualified for the game mode(s) in question, as stated in the [Beatmap Nominator Rules](https://osu.ppy.sh/help/wiki/People/Beatmap_Nominators/Rules).
+
+Each beatmap veto must be discussed by both parties - the mapper as well as the Nominator placing the beatmap veto - in an attempt to reach an agreed compromise.
+
+In cases where discussion does not take place or is inconclusive, members of the [Quality Assurance Team](https://osu.ppy.sh/groups/7)’s disqualification branch will mediate to either uphold or dismiss the veto.
+
+## Performing a beatmap veto
+
+As a Beatmap Nominator, you can veto a beatmap in one of two ways: 
+
+* On older beatmaps with forum threads, place a Bubble Pop icon
+* On newer beatmaps with discussions, post a Problem stamp
+
+In both cases, the beatmap veto should be stated as such, and should be accompanied by a detailed explanation as to why you believe the beatmap in question should not be considered for Qualification.
+
+A proper discussion which attempts to reach a mutual agreement by both sides of the argument must then follow.
+
+Assuming the issues are satisfactorily resolved or justified with all parties in agreement, the Nominator may lift the beatmap veto and allow Qualification to proceed.
+
+## What happens if no agreement is reached?
+
+Mapping is extremely diverse and creative differences are to be expected. This is to be encouraged for the sake of creating a variety of content all players can enjoy! However, such differences can naturally lead to disagreements.
+
+Where neither party can agree to compromise on an issue, the Quality Assurance Team will act as mediators to decide if the veto should be dismissed or upheld.
+
+If discussion is no longer progressing after 48 hours have passed, the mapper or Nominator(s) can request for mediation. If no request is received, this mediation will begin after one week, regardless of whether discussion is still ongoing or not.
+
+During mediation, the disqualification branch of the QAT will discuss the beatmap and reasons for and against the veto. This includes an evaluation as to whether the proposed changes are deemed necessary and/or beneficial. Mediation will last for no longer than three days.
+
+A decision to uphold or dismiss will be based on majority vote, with a minimum of either seven branch members, or half of the total members participating.
```

Comment:
is this the same for all modes

Reply:
at present, yes - it's planned for everyone to have a say in mediation but this might be changed to a per-mode basis if we struggle to get people contributing for all modes